cylc/flow/scheduler.py

```diff
@@ -643,7 +643,7 @@ async def run(self):
             await self.configure()
             await self.start_servers()
             await self.log_start()
-        except Exception as exc:
+        except (KeyboardInterrupt, asyncio.CancelledError, Exception) as exc:
             await self.shutdown(exc)
```

Comment:
Might want to copy over the try/except to ensure that shutdown traceback gets dumped to the flow log.

Reply:
@MetRonnie two approvals, but will let you reply this one before I merge it (happy if you or @oliver-sanders merge it after you've taken a look on this one too :+1: )